tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml

```diff
@@ -0,0 +1,20 @@
+---
+addr: '0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874'
+decimals: 18
+description: >- Tokenized 1 Troy Ounce Gold Bar, Redeemable in Precious Metal (Gold)
+  # The following options were found                                                                       #
+  # You may need to edit them and you MUST delete commented lines                                          #
+  # From the website: description meta tag                                                                 #
+  Veritaseum enables software-driven P2P capital markets without brokerages, banks or traditional exchanges
```

Comment:
The template comments (lines that start with `#`) and the auto-generated description should be deleted.

Reply:
delete the template comments